projects/RabbitMQ.Client/client/impl/Frame.cs

```diff
@@ -246,38 +251,79 @@ private static void ProcessProtocolHeader(Stream reader, ReadOnlySpan<byte> fram
             }
         }
 
-        internal static InboundFrame ReadFrom(Stream reader, byte[] frameHeaderBuffer, uint maxMessageSize)
+        internal static async ValueTask<InboundFrame> ReadFromPipeAsync(PipeReader reader, uint maxMessageSize)
```

Comment:
Why is the async suffix used here and not in other async methods?

Reply:
I will ensure this is consistent.

Reply:
https://github.com/rabbitmq/rabbitmq-dotnet-client/issues/1296